ocaml/xapi/xapi_host.ml

```diff
@@ -2810,4 +2814,5 @@ let apply_updates ~__context ~self ~hash =
       ~doc:"Host.apply_updates" ~op:`apply_updates
     @@ fun () -> Repository.apply_updates ~__context ~host:self ~hash
   in
+  Db.Host.set_last_software_update ~__context ~self ~value:(now_rfc3339 ()) ;
```

Comment:
Won't this update the field for the coordinator only? this needs to happen for supporters as well

Reply:
My understanding is this updates the entry for any host (not just the coordinator) on the coordinator.

Reply:
y confusion is due to the name of the host parameter which is called `~self`, this can be different from the host where the function is executed